main.py

```diff
@@ -20,6 +20,7 @@
 import pytz
 
 import discord
+from discord import utils as dutils
```

Comment:
what's this? why the `dutils`?
I'm generally not a fan of changing valid python just for pyright...